src/router/routes.js

```diff
@@ -4,7 +4,12 @@ const routes = [
     path: '/',
     component: () => import('layouts/MainLayout.vue'),
     children: [
-      { path: '', component: () => import('pages/Index.vue') }
+      { path: '', component: () => import('pages/Index.vue') },
```

Comment:
colocar um name pra essa página, e deixar no padrão, lembrando q vai utilizar form caso a página de create/edit for a mesma
```
pages/nameList.vue

pages/nameCreate.vue
pages/nameEdit.vue
       ou
pages/nameForm.vue
```